backend/apps/matching/templates/map/map.html

```diff
@@ -17,8 +17,8 @@
       mapViewContainerId: 'mapContainer',
       aURL: "{% url 'participant_JSON' 'A' %}",
       bURL: "{% url 'participant_JSON' 'B' %}",
-      aListURL: "{% url 'participant_list' 'A' 'COUNTRYCODE' 'PLZ' 0 %}",
-      bListURL: "{% url 'participant_list' 'B' 'COUNTRYCODE' 'PLZ' 0 %}",
+      aListURL: "{% url 'participant_list' 'A'%}?location_country_code=COUNTRYCODE&location_zipcode=PLZ&location_distance=0",
+      bListURL: "{% url 'participant_list' 'B'%}?location_country_code=COUNTRYCODE&location_zipcode=PLZ&location_distance=0",
```

Comment:
Is this really the django way to do it?

Reply:
I tried googleing it already and apparently it is.
other way is passing a dict with get params to urlencode.